src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -14,25 +42,22 @@ import { TemplateBaseComponent } from "../base";
   styleUrls: ["./button.component.scss"],
 })
 export class TmplButtonComponent extends TemplateBaseComponent implements OnInit {
-  /** TEMPLATE PARAMETER: "style" */
-  style:
-    | "information"
-    | "navigation"
-    | "full"
-    | "flexible"
-    | "medium"
-    | "short"
-    | "tall"
-    | "standard"
-    | "alternative" = "information";
+  /** TEMPLATE PARAMETER: "variant" */
+  variant: IButtonParams["variant"] = "information";
+  /** TEMPLATE PARAMETER: "style". Legacy, use "variant" instead. */
+  style: IButtonParams["style"];
   /** TEMPLATE PARAMETER: "disabled". If true, button is disabled and greyed out */
-  disabled: boolean = false;
+  disabled: IButtonParams["disabled"] = false;
   /** TEMPLATE PARAMETER: "text_align" */
-  textAlign: "left" | "centre" | "right" = "left";
+  textAlign: IButtonParams["textAlign"] = "left";
   /** TEMPLATE PARAMETER: "button_align" */
-  buttonAlign: "left" | "centre" | "right" = "centre";
+  buttonAlign: IButtonParams["buttonAlign"] = "centre";
   /** TEMPLATE PARAMETER: "icon". The path to an icon asset */
-  icon: string;
+  icon: IButtonParams["icon"];
+  /** TEMPLATE PARAMETER: "image_asset". The path to an image asset */
+  image: IButtonParams["image"];
+  /** @ignore */
+  variantIsCardPortrait: boolean = false;
```

Comment:
nit(blocking) adding extra variables to track individual variants might not scale very well. Probably better to include a local `variantMap` variable that splits the string, populates as `{portrait: true}` along with any other variants and use as a lookup in the code